Bottom: e1e7b50d41173b9b41d90ddf8326e31b0f2ded35
Top:    0f0d52ee03c22dd0bceed6efad415a9370a150c5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2016-02-18 17:38:12 +0000

Add warning about bogus code in strflags.c


---

diff --git a/src/strflags.c b/src/strflags.c
index 3e57696..d6d6273 100644
--- a/src/strflags.c
+++ b/src/strflags.c
@@ -576,6 +576,7 @@ dump_flag (FlagType * f)
 {
   int l;
   printf ("F:%08x T:[", f->f);
+#warning THIS NEXT LINE IS BOGUS, THE ARRAY IS HALF THIS LENGTH
   for (l = 0; l < (MAX_LAYER + 7) / 8; l++)
     printf (" %02x", f->t[l]);
   printf ("]");
